src/blocks/card.css

```
@@ -0,0 +1,83 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
```

```suggestion
  padding: 32px 16px;
```
One 16px statement should be enough

---

fixed!